docs/changelog.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_label: 'Changelog'
+---
+
+# Changelog
+
+## 0.0.11
+
+:::info
+
+This version is a release preview. Our product is in beta and we would love for developers to try it out and provide feedback! Contact
+us at info@regression.gg for dedicated support and access.
+
+:::
+
+**New Platform Features**
+
+- [Agent Builder](/players/creating-bots/agent-builder) Beta - Our GPT-enabled agent builder product is live for select players, now with improved validation tools. If you&apos;d like to try it out, let us know!
+- Our documentation site is now live! This docs site includes API/SDK references, tutorials, guides, and FAQs for both players and developers.
+- [API Keys](/studios/api_reference/authentication) - developers and players can now create API keys to interact with RG services
+- New onboarding flow for players and game developers
+- Logs for bots can now be downloaded - available via API or through the [Bot History dashboard](/studios/unity/unity-sdk/in-editor-replay)
```

Comment:
This should be under Unity SDK Features since it's only available for Studios... we may need a different solution for players.

Reply:
Moved!

Reply:
players can download their logs right after the match finishes before returning to lobby, so we have 'something' today to meet their needs